src/napari_workflows/_io_yaml_v1.py

```diff
@@ -12,7 +12,7 @@ def save_workflow(filename:str, workflow:Workflow):
     # Filter out workflow steps that do not represent a processing step
     workflow_to_save = Workflow()
     for key, value in workflow._tasks.items():
-        if not isinstance(value[0], np.ndarray):
+        if not hasattr(value[0], "shape"): 
```

Comment:
```suggestion
        if callable(value[0]): 
```
If we do it like this, it would be consistent with an other place: 
https://github.com/haesleinhuepf/napari-workflows/blob/main/src/napari_workflows/_workflow.py#L41